urbanairship-react-native/ios/UARCTModule/UARCTEventEmitter.m

```diff
@@ -216,6 +224,13 @@ - (void)showInboxMessage:(NSString *)messageID {
     [self sendEventWithName:UARCTShowInboxEventName body:body];
 }
 
+#pragma mark -
+#pragma mark Chat
+
+- (void)conversationUpdated {
+    [self sendEventWithName:UARCTConversationUpdatedEventName];
```

Comment:
is there supposed to be a body with this event?

Reply:
No I guess not, just the event